modules/gateway/persist.go

```diff
@@ -31,3 +33,13 @@ func (g *Gateway) load(filename string) (err error) {
 	}
 	return
 }
+
+// create logger
+// TODO: when is the logFile closed? Does it need to be closed?
```

Comment:
Only needs to be closed when the daemon is closed. I guess that means each module should get a Close() function in addition to the logger.